tests/unit/ast/test_pre_parser.py

```diff
@@ -191,6 +191,25 @@ def test_parse_pragmas(code, pre_parse_settings, compiler_data_settings, mock_ve
     assert compiler_data.settings == compiler_data_settings
 
 
+pragma_venom = [
+    """
+    #pragma venom
```

Comment:
can we add a test for when both are set?

Reply:
forgot to commit the file, sorry! should be there now

Reply:
thanks!